@types/node pinned to ^12 breaks Typescript builds dependent on later major Node.js versions/typings

### Problem description

When including grpc-js in a Typescript project built for a major Node.js version later than 12, the explicit pin of "^12.12.47" forces inclusion of an extra set of `@types/node` typings.  This leads to the dreaded `...error TS4090: Conflicting definitions for 'node' found at...`.

### Reproduction steps

Create a Typescript / Node.js project which requires Node.js typings for Node 13 or above.  Compile said project and bask in the glory/pain of multiple included conflicting typings.

### Environment
 - OS name, version and architecture: [e.g. Linux Ubuntu 18.04 amd64]
 Any
 - Node version [e.g. 8.10.0]
13+
 - Node installation method [e.g. nvm]
nvm
 - Package name and version [e.g. gRPC@1.12.0]
 @grpc/grpc-js@1.2.5

### Additional Comments
Switching to `>=12.12.47` would fix this issue if `grpc-js` is / can be made compatible with later typings.

--- a/packages/grpc-js/package.json
+++ b/packages/grpc-js/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@grpc/grpc-js",
-  "version": "1.2.5",
+  "version": "1.2.6",
   "description": "gRPC Library for Node - pure JS implementation",
   "homepage": "https://grpc.io/",
   "repository": "https://github.com/grpc/grpc-node/tree/master/packages/grpc-js",
@@ -57,7 +57,7 @@
     "posttest": "npm run check"
   },
   "dependencies": {
-    "@types/node": "^12.12.47",
+    "@types/node": ">=12.12.47",
     "google-auth-library": "^6.1.1",
     "semver": "^6.2.0"
   },

